### DISCUSSION TITLE
Vodafone Italia blocking mastodon.social

### DISCUSSION THREAD

**Discussion by RGB-es** [mastodon/mastodon#19384] [General]
Hello, all!

Vodafone Italia, for their home customers using their _Vodafone station_, have a "feature" called _rete sicura_ that, in principle, "protect" their users. Since a couple of days, rete sicura is blocking mastodon.social, so I cannot enter my account from home. I'm trying to contact them to report the problem, but I'm not sure how good they are at listening.

Regards

**Comment by RGB-es** [comment]
Reporting the issue was impossible: the site always crashed. It was also impossible to set the service to do what I want. So finally I was able to completely disable it. Now, for me everything is working now, but the problem remains, I guess: vodafone rete sicura family is still blocking mastodon.social.

**Reply by gilgongo** [reply]
You could try the abuse contact form for your IP address perhaps? https://www.ripe.net/ although that's usually reserved for reports about traffic coming out of the address you might tell them that you can't report it any other way.

**Reply by RGB-es** [reply]
@gilgongo I ended up disabling the service, so I don't have access to the  service's report site anymore.
